code/game/objects/structures/arclumin.dm

```diff
@@ -8,11 +8,14 @@ but we are all coders together.
 
 Goodbye, man. We'll miss you.
 
+This memorial has been designed for him and any future coders to perish.
```

Comment:
You better watch out, you better not shout, cause the maintainers are coming to town